packages/block-library/src/file/index.js

```diff
@@ -200,33 +201,71 @@ export const settings = {
 			showDownloadButton,
 			downloadButtonText,
 		} = attributes;
-
-		return ( href &&
+        return ( href &&
 			<div>
-				{ ! RichText.isEmpty( fileName ) &&
-					<a
-						href={ textLinkHref }
-						target={ textLinkTarget }
-						rel={ textLinkTarget ? 'noreferrer noopener' : false }
-					>
-						<RichText.Content
-							value={ fileName }
-						/>
-					</a>
-				}
-				{ showDownloadButton &&
-					<a
-						href={ href }
-						className="wp-block-file__button"
-						download={ true }
-					>
-						<RichText.Content
-							value={ downloadButtonText }
-						/>
-					</a>
-				}
+                { fileName !== null &&
+				<a
+					href={ textLinkHref }
+					target={ textLinkTarget }
+					rel={ textLinkTarget ? 'noreferrer noopener' : false }
+				>
+					<RawHTML>{ fileName }</RawHTML>
```

Comment:
I'm not sure I understand what it is about changing from RichText to PlainText which would require that the `save` function be updated.

Furthermore, can you explain why `RawHTML` is necessary here, rather than just inserting the `fileName` verbatim (without it)?